src/EPR.RegulatorService.Facade.Core/TradeAntiVirus/AntivirusService.cs

```diff
@@ -25,7 +25,7 @@ public async Task<HttpResponseMessage> SendFile(SubmissionType submissionType, G
                 Key = fileId,
                 Extension = Path.GetExtension(fileName),
                 FileName = Path.GetFileNameWithoutExtension(fileName),
-                Collection = GetCollectionName(submissionType.GetDisplayName()),
+                Collection = GetCollectionName(submissionType.GetDisplayName<SubmissionType>()),
```

Comment:
Did we mean to update this?

Reply:
Yes other wise I would be duplicating code